crates/router/src/connector/adyen.rs

```diff
@@ -797,13 +798,17 @@ impl api::IncomingWebhook for Adyen {
         db: &dyn StorageInterface,
         merchant_id: &str,
     ) -> CustomResult<Vec<u8>, errors::ConnectorError> {
-        let key = format!("whsec_verification_{}_{}", self.id(), merchant_id);
-        let secret = db
-            .get_key(&key)
-            .await
-            .change_context(errors::ConnectorError::WebhookVerificationSecretNotFound)?;
-
-        Ok(secret)
+        let key = conn_utils::get_webhook_merchant_secret_key(self.id(), merchant_id);
```

Comment:
Since this logic is redundant in all the connectors, can we have it as the default implementation?

Reply:
I think it's done here -> https://github.com/juspay/hyperswitch/pull/1331/files#diff-8f740d4eb7ef77de424329a12be7f7a12b09fb805a6551d25e7d8edbda606ddf @vspecky

Reply:
@sai-harsha-vardhan Yes, so I think we can avoid blocking this PR on the default implementation as opposed to doing the implementation again here.